entities/scheduledcampaigns.go

```diff
@@ -0,0 +1,15 @@
+package entities
+
+import (
+	"time"
+
+	"github.com/segmentio/ksuid"
+)
+
+type ScheduledCampaign struct {
```

Comment:
I'm thinking this is a better name.

```suggestion
type CampaignSchedule struct {
```